### ISSUE TITLE
Audio doesn't seem to be working

### ISSUE THREAD

**Issue by bobbyl140** [rileytestut/Delta#273]
Hi,
I built Delta from source a few weeks ago (I don't remember when exactly but it was after the last commit on October 19th. I installed it (using AltStore) and audio doesn't play at all, no matter the respect ring switch setting. Does this happen to anyone else? Thank you!

**Comment by bobbyl140** [comment]
This problem still seems to exist with the latest AltStore version. Is there any testing I can do to encourage a fix?

**Comment by Alex1478** [comment]
Same issue here! no sound 🥲
iPhone 15 Pro Max
iOS 17.3
Delta: 1.4.2

**Comment by bobbyl140** [comment]
@Alex1478 I ended up building it from source and the resulting app does have sound. So I guess it’s fixed but not released yet

**Comment by Alex1478** [comment]
Just compiled the app, version 1.5 build 80
Unfortunately, same issue! I confirm that the mobile device isn't muted

**Comment by bobbyl140** [comment]
Well I know mine worked the other day, but it no longer is! Don’t know what that’s about… So yeah, the bug still exists.

**Comment by Alex1478** [comment]
Thank you for your help! I will try on another device and see

**Comment by bobbyl140** [comment]
Yeah no problem! I did try on an iPad and my MacBook with the same result if it saves you any hassle.

**Comment by bobbyl140** [comment]
@rileytestut is there any news on this? I see that Delta received a new beta today, but the changelogs don't mention this issue. 
@Alex1478 and anyone else interested, I've been using older builds of Delta for now, specifically this one [here](https://github.com/ianclawson/Delta-iPac-Edition) because the icon is different. I could change it myself but that was easier to do quickly.

**Comment by adrianmihalko** [comment]
1.5 beta has no sound

**Comment by Rosalina121** [comment]
I've found that turning off the Sound Recognition in Accessibility settings fixes the issue.

**Comment by creatorKoo** [comment]
> I've found that turning off the Sound Recognition in Accessibility settings fixes the issue.

This comment saved my life, thank you

**Comment by creatorKoo** [comment]
> > I've found that turning off the Sound Recognition in Accessibility settings fixes the issue.
> 
> This comment saved my life, thank you

Delta app needs to fix that problem

**Comment by bobbyl140** [comment]
> I've found that turning off the Sound Recognition in Accessibility settings fixes the issue.

First of all, how on earth did you figure that out? (Not a serious question). 
This almost makes me think it’s an iOS problem, as I don’t believe apps can change functionality based on system settings such as that. Don’t hold me to it though.

**Comment by creatorKoo** [comment]
> > I've found that turning off the Sound Recognition in Accessibility settings fixes the issue.
> 
> First of all, how on earth did you figure that out? (Not a serious question.) This almost makes me think it’s an iOS problem, as I don’t believe apps can change functionality based on system settings such as that. Don’t hold me to it though.

I completely agree. However, other apps don't have this problem at all, so why does only the Delta app have this problem?

**Comment by bobbyl140** [comment]
Fair point. Maybe it’s a combination, I really don’t know. Just thought insight might be useful to someone at some point.

**Comment by princhcanal** [comment]
i had the same issue and i was able to fix it by just updating to the latest version if iOS (17.4.1)

**Comment by creatorKoo** [comment]
> i had the same issue and i was able to fix it by just updating to the latest version of iOS (17.4.1)

I also use the latest version of iOS(17.4.1) but still have the problem

**Comment by bobbyl140** [comment]
I’m on the iOS 17.5 beta and it worked earlier today but isn’t right now.

**Comment by bobbyl140** [comment]
> Funny enough, I’m no longer having the problem with the App Store build.

^

**Comment by fredsvanelli** [comment]
I’ve tried every fix I found on the internet and still no sound:

* Delta from App Store v1.5.3
* iPhone 14 Pro Max
* iOS 17.4.1
* Physical mute button: Sound enabled
* Volume: Max
* Delta > Respect Silent Mode: OFF
* Accessibility > Sound recognition: OFF
* Accessibility > Voice over: OFF
* Sound and haptics > Adjust with button: ON

On iPad with the same iOS version it works fine.

On my friend’s iPhone with the exact same configs, it works fine.

Sometimes it works on my iPhone all of the sudden for some minutes, then I switch between apps and when I’m back to Delta, no sound again for days.

If I reboot the iPhone, sometimes the sound works, some times it doesn't. And even when it works, it stops when I open other apps.

I've tried closing all other apps. Didn't work.

I've also tried uninstalling Delta and installing it again. Also didn't work.

**Comment by fredsvanelli** [comment]
> I’ve tried every fix I found on the internet and still no sound:
> 
> * Delta from App Store v1.5.3
> * iPhone 14 Pro Max
> * iOS 17.4.1
> * Physical mute button: Sound enabled
> * Volume: Max
> * Delta > Respect Silent Mode: OFF
> * Accessibility > Sound recognition: OFF
> * Accessibility > Voice over: OFF
> * Sound and haptics > Adjust with button: ON
> 
> On iPad with the same iOS version it works fine.
> 
> On my friend’s iPhone with the exact same configs, it works fine.
> 
> Sometimes it works on my iPhone all of the sudden for some minutes, then I switch between apps and when I’m back to Delta, no sound again for days.
> 
> If I reboot the iPhone, sometimes the sound works, some times it doesn't. And even when it works, it stops when I open other apps.
> 
> I've tried closing all other apps. Didn't work.
> 
> I've also tried uninstalling Delta and installing it again. Also didn't work.

I just found the problem. The app "Rise Sleep", even closed (and killed), was running some background process that was locking the audio output only from Delta. I uninstalled it and now it's working!

**Comment by bobbyl140** [comment]
I’ve never used that app, yet I have the issue. So I’m guessing other apps can also “lock” the now playing state.
